blocks/quote/quote.css

```diff
@@ -7,6 +7,15 @@
   position: relative;
 }
 
+.block.quote.white {
```

Comment:
There's a `white` mixin already, would that work?

Reply:
Hello @dnbute it doesn't work for the :before and :after elements.